src/main/resources/data-repository-openapi.yaml

```diff
@@ -2658,6 +2692,56 @@ definitions:
         items:
           $ref: '#/definitions/PolicyModel'
 
+  DataDeletionRequest:
```

Comment:
Should this be named more specifically to mention tables? Or will file deletions eventually shift over to use this model, too?

Reply:
I imagine the file model will be different, updating.